CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- `MessagingSession` infinite reconnect loop to break on timeout.
+- `MessagingSession` `getMessagingSessionEndpoint` call is now backwards compatible with aws js sdk v2 and lower.
```

Comment:
nit: aws js sdk v2 -> AWS JS SDK v2. 

Should we drop "and lower"? Not sure if we support it.

Reply:
done